R/awsjavasdk-internal.R

```diff
@@ -17,8 +17,9 @@
 .app_dir <- rappdirs::app_dir("awsjavasdk", version = packageVersion("awsjavasdk"))
 
 .onAttach <- function(libname, pkgname) {
+  set_sdk_file_root()
   if (!aws_sdk_present()) {
-    packageStartupMessage("awsjavasdk did not detect its own copy of the AWS SDK; you must execute install_aws_sdk() in order to download the SDK and for the package to work as expected")
+    packageStartupMessage(paste0("awsjavasdk did not detect its own copy of the AWS SDK. You must execute install_aws_sdk() in order to download the SDK and for the package to work as expected. If you've previously installed to a non-standard location, call set_sdk_file_root({{your alt_install root}}, confirm = TRUE) to point this package to that location and then run awsjavasdk::load_sdk()"))
```

Comment:
@daroczig I can't speak as to whether CRAN wants to host such a package - so take heart!  However, space certainly isn't a concern per se (if updated rarely), and others have hosted Apache Licensed JARs there before.  So, you should be fine to pursue that route in theory.  The rub is in practice.  How up-to-date do you want the jars to be?

AWS releases patches at an astonishing pace (https://aws.amazon.com/releasenotes/).  I don't expect that CRAN wouldn't tolerate a non-huge package publishing at that rate, much less a large one (because they state the advantage of a large jar-only packages is that they can be updated rarely).  Maybe you have an idea about how to balance those requirements?

Regardless, this was a pretty fast-and-dirty approach to get something out the door so I could build other things.  I welcome improvements, or indeed replacement if you come up with something great. :)